### ISSUE TITLE
Enforce Permissions

### ISSUE THREAD

**Issue by chris-miaskowski** [stoplightio/stoplight.io#1027]
*NOTE: Please work SD-918 before this ticket.* 

Design docs: https://github.com/stoplightio/design-docs/blob/master/docs/external-services.md#org-and-project-visibility.

*User Story*

As a Stoplight User I want permissions to be enforced so that users cannot perform actions or see information that they shouldn't.

 

*Acceptance Criteria and Scope*

* Lock down the corresponding functions to the permissions below:
** ci:read: allows users to view project ci tokens and the instructions on how to use them during the analyze process
** ci:write: allows users to re-generate project ci tokens
** org:read: allows users to view content belonging to the org
** org:write: allows users to change org settings
** org:delete: allows users to delete an org (and all of its projects) from Stoplight
** org:push: allows users edit content belonging to org
** billing:write: allows users modify billing information
** billing:read: allows users to view billing information
* hide/disable any UI elements corresponding to the permissions above
* check permissions on api endpoints and fail with appropriate message if not authorized
* Repo provider story needs to be completed prior 

*Out of Scope*

* implementing any functionality that has not been implemented yet, but corresponds to a permission above. For example, ci:write allow users to re-generate ci tokens. We do not have that functionality implemented yet, so this story does not cover the implementation of that functionality.

* UI permission checks will be done in another story

**Comment by tbarn** [comment]
@StefanDywersant was this the right repo to make this? This is just the static site. I can't see the design doc, so I don't have full context though.

**Comment by XVincentX** [comment]
Yeah I believe this is wrong.

**Comment by lottamus** [comment]
Closing. @chris-miaskowski can you move this to the API repo?
